libs/external-router/driver/testing/src/config.ts

```diff
@@ -0,0 +1,24 @@
+import { InjectionToken } from '@angular/core';
+
+import { DaffExternalRouterType } from '@daffodil/external-router';
+
+/**
+ * The configuration for the testing driver. It is a dictionary of "url": "type",
+ * pairs, where the URL is a URL-safe routeable path, e.g. "sweatshirts" or "t-shirts"
+ * and the type is a known type to your application, e.g. "CATEGORY".
+ */
+export interface DaffExternalRouterDriverTestingConfig {
+	[url: string]: DaffExternalRouterType;
+}
+
+/**
+ * The token used by Daffodil to hold the driver's configuration.
+ *
+ * @docs-private
+ */
+export const DAFF_EXTERNAL_ROUTER_DRIVER_TESTING_CONFIG = new InjectionToken<
+	DaffExternalRouterDriverTestingConfig
+>('DAFF_EXTERNAL_ROUTER_DRIVER_TESTING_CONFIG', {
+	providedIn: 'root',
```

Comment:
providedIn is 'root' by default, so this can be removed

Reply:
https://github.com/angular/angular/blob/c461acd12e529a253b17ebfae41c4a81fb757707/packages/core/src/di/injection_token.ts#L64

Reply:
Oh, you're right. I missed that. I'd still like to keep the prop though, to make it explicitly clear.